docs/package_integration/uds-package-practices.md

```diff
@@ -120,12 +112,30 @@ In practice, this results in the following for the second release of a package f
 17.2.1-uds.1
 ```
 
-## General
+## Documenting Exceptions
+
+If you are unable to meet a requirement or best practice, document the justification in your package repository `docs/package_exceptions.md`
+
+Some examples of valid justifications would be:
+- ✅ The last step of deployment requires a click-ops interaction and cannot be automated through IaC
```

Comment:
```suggestion
- ✅ The last step of SSO configuration requires a click-ops interaction and cannot be automated declaratively
```

Reply:
Not sure if this is actually a valid exception given that this is a **Must** as I am interpreting it - as written the reqs _only_ require integration with the UDS Operator (so SSO, Networking, and Monitoring) so only these configurations **must** be declarative - if something falls outside of these configurations it wouldn't matter and would not need to be documented here (i.e. the final Mattermost setup of a Team for example) - and if an application didn't integrate with one of the UDS Operator features I do think that it would be worth holding it back a level intentionally and if it is important working with upstream to fix it.